packages/io-page/io-page.2.3.0+dune/opam

```diff
@@ -23,5 +23,5 @@ IO pages are page-aligned, and wrapped in the `Cstruct` library to avoid
 copying the data contained within the page.
 """
 url {
-  src: "git://github.com/mirage/io-page.git"
```

Comment:
This feels suspect to me, that does not seem to package 2.3.0 but rather the git default branch

I looked at https://github.com/mirage/io-page/tree/v2.3.0 and it seems to work with dune, so does seemingly https://opam.ocaml.org/packages/io-page/io-page.2.3.0/ hence the whole port leaves me with a bit of head-scratching what is going on here.

Reply:
That's a good point! Can you open an issue so we keep track of this and investigate it further without blocking this PR?

Reply:
Sure: https://github.com/dune-universe/opam-overlays/issues/142

The PR is approved anyway, since this is not the place to investigate io-page and it fixes existing issues.